apps/server/src/app/routes/users.router.ts

```diff
@@ -276,6 +286,7 @@ router.get(
     })
 )
 
+// TODO: Remove this endpoint or refactor to work with new Auth
```

Comment:
Are these comments relevant? I don't understand their purpose.

Reply:
It's purely for me to track things down as I continue working on this. I wanted to share a draft to get feedback on overall direction. They'll be removed as the changes are made.